Parsers/ASimAuditEvent/Tests/Barracuda_WAF_ASimAuditEvent_SchemaTest.csv

```diff
@@ -0,0 +1,98 @@
+﻿Result
+"(1) Warning: Missing recommended field [ActorUsername]"
```

Comment:
AdminName_s is present and not mapped to ActorUsername. Please map ActorUsernameType and ActorUserType as well.

Reply:
Done